app/sections/view-capabilities/macro.njk

```diff
@@ -36,20 +36,20 @@
                 innerComponent: viewDataText({
                   dataTestId: 'view-question-data-text-description-paragraph-one',
                   data: 'Capabilities are a set of requirements that are defined using short descriptions called epics. Capabilities are achieved when a Catalogue Solution meets the required epics.',
-                  classes: 'nhsuk-u-margin-bottom-5 nhsuk-grid-column-two-thirds'
+                  classes: 'nhsuk-u-padding-left-0 nhsuk-u-padding-right-0 nhsuk-u-margin-bottom-5 nhsuk-grid-column-two-thirds'
```

Comment:
do we need to specify these if they are padding left and right 0?

Reply:
they both cancel out the padding that the class nhsuk-grid-column-two-thirds adds